pgml-docs/docs/gym/announcement.md

```diff
@@ -0,0 +1,70 @@
+---
+hide:
+  - navigation
+  # - toc
+  - comments
+---
+
+# Announcing the PostgresML Gym!
+
+Today we're thrilled to announce the launch of our brand new offering: a **free tier** of hosted PostgresML for machine learning enthusiasts and hobbyists! We affectionatly call it "the gym" as its goal is to train one's ML muscle with practical tutorials touching common topics such as generic regression, classification and fraud prediction.
+
+Sign up takes just a few seconds and you'll get access to your own personal PostgresML instance.
+
+<br />
+
+<p align="center" markdown>
+  [Sign Up](https://gym.postgresml.org/signup/){ .md-button .md-button--primary .md-button }
+</p>
+
+
+## Notebooks for Postgres
+
+One big piece of novelty we think is exciting is our freshly designed Notebooks. They work much like the Jupyter Notebooks we love and use with Python, except ours are tailored towards building production-ready models with SQL.
```

Comment:
I don't think this is a major selling point. Right now, our notebooks are arguably weaker than Jupyter.